core/merginapi.cpp

```diff
@@ -1351,14 +1351,27 @@ QString MerginApi::extractServerErrorMsg( const QByteArray &data )
   if ( doc.isObject() )
   {
     QJsonObject obj = doc.object();
-    QJsonValue vDetail = obj.value( "detail" );
-    if ( vDetail.isString() )
+    if ( obj.contains( QStringLiteral( "detail" ) ) )
     {
-      serverMsg = vDetail.toString();
+      QJsonValue vDetail = obj.value( "detail" );
+      if ( vDetail.isString() )
+      {
+        serverMsg = vDetail.toString();
+      }
+      else if ( vDetail.isObject() )
+      {
+        serverMsg = QJsonDocument( vDetail.toObject() ).toJson();
+      }
     }
-    else if ( vDetail.isObject() )
+    else if ( obj.contains( QStringLiteral( "name" ) ) )
     {
-      serverMsg = QJsonDocument( vDetail.toObject() ).toJson();
+      QJsonArray errors = obj.value( "name" ).toArray();
```

Comment:
What would happen if `name` is not an array?

Reply:
Do we have responces where `name` is not an array?

Reply:
I do not know what Flask is capable of :) Best to double-check